### PR TITLE
CI: Fix GH invalid workflow error "A sequence was not expected"

### PR DESCRIPTION
**Pull request by zakkak**
Fixes GH workflow which now fails with:

```
Invalid workflow file: .github/workflows/main.yml#L16
The workflow is not valid. .github/workflows/main.yml (Line: 16, Col: 22): A sequence was not expected
```

Source: https://github.com/oracle/graal/actions/runs/2244856633

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ on:
       - '.travis.yml'
       - '.github/workflows/quarkus.yml'
       - '**.md'
-  workflow_dispatch: []
+  workflow_dispatch:
```
Comment (dougxc) [inline .github/workflows/main.yml:16]: What's the point of an empty event?

Comment (dougxc) [reply .github/workflows/main.yml:16]: Thanks. A comment to explain that would be great.

Comment (zakkak) [reply .github/workflows/main.yml:18]: Done

**Comment by dougxc** [comment]
I'll merge this as part of #4523.

**Pull request closed without merging**